java2typescript-jackson/src/test/resources/java2typescript/jackson/module/WriterPreferencesTest.enumToStringLiteralType.d.ts

```diff
@@ -3,5 +3,11 @@ export type Enum =
     | "VAL2"
     | "name";
 
+export interface TestClassHasMapWithEnumKey {
+    enumKeyMap: { [key in Enum ]: string;};
+    stringKeyMap: { [key: string ]: string;};
+    intKeyMap: { [key: string ]: string;};
```

Comment:
Yes. The only reason I want to keep at least `stringKeyMap` in this test is so that I know normal `Map<String, T>` are not affected/do not regress by this change.